tests/test_metrics/test_instance_seg.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+import unittest
+
+from mmeval.metrics import InstanceSeg
+
+
+class TestInstanceSegMetric(unittest.TestCase):
+
```

Comment:
Suggesting using pytest to unify the test style~

Reply:
fixed